cookies/http-state/resources/cookie-setter.py

```diff
@@ -48,25 +48,25 @@ def body_with_expectation(self):
     """Returns a full HTML document which contains the expectation."""
     html_frame = dump_file(self.__resources_dir, EXPECTATION_HTML_SCAFFOLD)
     expected_data = dump_file(self.__test_files_dir, self.__expectation_file)
-    return html_frame.format(**{u'data' : expected_data})
+    return html_frame.replace(b"{data}", expected_data)
 
 def find_all_test_files(root):
   """Retrieves all files from the test-files/ folder and returns them as
   string pair as used in the JavaScript object. Sorted by filename."""
   all_files = []
-  line_template = u'{{file: "{filename}", name: "{filename}"}},'
+  line_template = b'{{file: "{filename}", name: "{filename}"}},'
   for file in listdir(path.join(root, DEFAULT_RESOURCE_DIR, DEFAULT_TEST_DIR)):
     if file.endswith(u'-test'):
-      name = file.replace(u'-test', u'')
-      all_files.append(line_template.format(**{u'filename' : name}))
+      name = file.replace(u'-test', u'').encode()
+      all_files.append(line_template.replace(b"filename", name))
```

Comment:
Shouldn't this be

```python
      all_files.append(line_template.replace(b"{filename}", name))
```
?

Also, I'd prefer to keep the template string as text and do the encoding here, e.g.

```python
      name = file.replace(u'-test', u'').
      all_files.append(line_template.format(filename=name).encode())
```

Reply:
Update. Thank you!